core-bundle/tests/Intl/LocalesTest.php

```diff
@@ -154,7 +154,7 @@ public function testGetsLocaleLabels(): void
 
         $this->assertSame(
             [
-                'gsw_Hans_AT' => 'Schwiizertüütsch (Veräifachti Chineesischi Schrift, Ööschtriich)',
+                'gsw_Hans_AT' => 'Schwiizertüütsch (Veräifachti Chineesischi Schrift, Ööschtriich) - Schweizerdeutsch (Vereinfacht, Österreich)',
                 'de_CH' => 'Tüütsch (Schwiiz) - Deutsch (Schweiz)',
             ],
             $this->getLocalesService()->getDisplayNames(['gsw_Hans_AT', 'de_CH'], 'gsw', true)
```

Comment:
```suggestion
                'gsw_Hans_AT' => 'Schwiizertüütsch (Veräifachti Chineesischi Schrift, Ööschtriich)',
                'de_CH' => 'Tüütsch (Schwiiz) - Deutsch (Schweiz)',
            ],
            $this->getLocalesService()->getDisplayNames(['gsw_Hans_AT', 'de_CH'], 'gsw_Hans_AT', true)
```
This fix should work on both systems.

Reply:
```suggestion
                'gsw_Latn_AT' => 'Schwiizertüütsch (Latiinisch, Ööschtriich)',
                'de_CH' => 'Tüütsch (Schwiiz) - Deutsch (Schweiz)',
            ],
            $this->getLocalesService()->getDisplayNames(['gsw_Latn_AT', 'de_CH'], 'gsw', true)
```